pages/v/[id]/_middleware.ts

```diff
@@ -0,0 +1,22 @@
+import { NextResponse } from 'next/server';
+import type { NextRequest } from 'next/server';
+
+const COOKIE_KEY = 'streamPlayerType';
+const SPLIT_PERCENTAGE = 0.1;
```

Comment:
Lets start at 0.2 YOLO.

Also could move this to an env var. Env vars are accessible in Next middleware, right?